sui/p2p_transfer/index.js

```diff
@@ -19,48 +20,49 @@ const main = async () => {
   const SENDER_PRIVATE_KEY = process.env.ACC1_PRIVATE_KEY;
   const sender_keypair = getKeyPairFromExportedPrivateKey(SENDER_PRIVATE_KEY);
 
-  const RECIEVER_PRIVATE_KEY = process.env.ACC2_PRIVATE_KEY;
-  const receiver_keypair = getKeyPairFromExportedPrivateKey(RECIEVER_PRIVATE_KEY);
-  const receiver_address = receiver_keypair.getPublicKey().toSuiAddress();
-
 
   // create a new SuiClient object pointing to the network you want to use
   let url = getFullnodeUrl('mainnet');
   if (URL_OVERRIDE) {
       url = URL_OVERRIDE;
   }
   const suiClient = new SuiClient({ url: url });
+  const gasPrice = await suiClient.getReferenceGasPrice()
+  let gasCoin = null
 
   while (true) {
     try {
       const txb = new TransactionBlock();
       const [coin] = txb.splitCoins(txb.gas, [txb.pure(1)]);
-      txb.transferObjects([coin], receiver_address);
+      txb.transferObjects([coin], sender_keypair.toSuiAddress());
```

Comment:
fair enough, I'll add back in the second account.  I just wanted to make it easier to run the tests, it doesn't change performance in sui

Reply:
I've added the second address back in, and also update the aptos version to use the correct environment variable for receiver so that it also uses a different address